packages/slice-machine/tests/components/UpdateVersionModal.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import "@testing-library/jest-dom";
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { render } from "@testing-library/react";
+import UpdateVersionModal from "../../components/UpdateVersionModal";
+
+const mockDispatch = jest.fn();
+jest.mock("react-redux", () => ({
```

Comment:
I think the best way of testing components is to do integration tests (as the redux documentation suggested). 

As we discussed on the tests strategy meeting, our priority number one is to spend all our effort on writing unit tests, because they are quicker to write and give quicker feedbacks.

So we have to stop spending time to test react components for the moment, the question is what we want to do for this test ? 
1) We can remove it 
2) We can reduce his scope by transforming it into a unit tests 
3) Other options ?

But for sure, we cannot keep a test that requires to be updated every time we changed the SM state just for testing a minor feature (the cost is heavy for a tiny gain).

What do you suggest ?